MeetupPlatformApi/DataTransferObjects/AuthenticationTokenPairOutputDto.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace MeetupPlatformApi.DataTransferObjects;
+
+using System.ComponentModel.DataAnnotations;
+
+public class AuthenticationTokenPairOutputDto
```

Comment:
Can't we name this DTO something like `TokenPairDto`, since it just contains 2 tokens?